examples/cards_fetch.py

```diff
@@ -25,11 +25,11 @@
 
 # fetch by card_uri
 cf = CardsFetch(account)
-card = cf.load(account, card_uri=card_uri)
-card.card_type # 'VIDEO_POLLS'
-card.id # '5g83p'
+card = cf.load(account, card_uris=[card_uri]).first
+print card['card_type']
```

Comment:
After making the changes to `creative.py` you'll have to update the accessors for any properties being printed, so:
`print['card_type']` now becomes `print.card_type`